docs/KubeflowMode.md

```diff
@@ -2,7 +2,7 @@
 ===
 Now NNI supports running experiment on [Kubeflow](https://github.com/kubeflow/kubeflow), called kubeflow mode. Before starting to use NNI kubeflow mode, you should have a kubernetes cluster, either on-prem or [Azure Kubernetes Service(AKS)](https://azure.microsoft.com/en-us/services/kubernetes-service/), a Ubuntu machine on which [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) is installed and configured to connect to your kubernetes cluster. If you are not familiar with kubernetes, [here](https://kubernetes.io/docs/tutorials/kubernetes-basics/) is a goot start. In kubeflow mode, your trial program will run as kubeflow job in kubernetes cluster. 
 
-## Prerequisite
+## Prerequisite for on-premises

```

Comment:
How about 'Prerequisite for on-premises Kubernetes Service'?

Reply:
Updated to  'Prerequisite for on-premises Kubernetes Service'